deploy/templates/azuredeploy.standard.json

```diff
@@ -317,10 +317,17 @@
             }
         },
         "appInsightsName": {
-            "type": "string",
-            "defaultValue": "[concat('appinsights-', take(uniqueString(subscription().subscriptionId, resourceGroup().id), 6))]",
-            "metadata": {
+              "type": "string",
```

Comment:
Please fix spaces which do not align.  Use visual studio code and arm extension and do a formatdocumemt if you want it fast.

Reply:
fixed